ArduCopter/Copter.cpp

```diff
@@ -445,6 +445,40 @@ bool Copter::has_ekf_failsafed() const
     return failsafe.ekf;
 }
 
+// get target location (for use by scripting)
+bool Copter::get_target_location(Location& target_loc)
+{
+    switch (flightmode->mode_number()) {
+    case Mode::Number::RTL:
+    case Mode::Number::SMART_RTL:
+    case Mode::Number::AVOID_ADSB:
+    case Mode::Number::GUIDED:
+    case Mode::Number::AUTO:
+    case Mode::Number::FOLLOW:
+        return flightmode->get_wp(target_loc);
+        break;
+    default:
+        break;
+    }
+    return false;
```

Comment:
I don't think you need the switch? You can just return the flight mode call? Looks like it will correctly return false if it is not implemented. 
```suggestion
    return flightmode->get_wp(target_loc);
```